debian/lyrionmusicserver.preinst

```diff
@@ -0,0 +1,39 @@
+#!/bin/sh
+
+# preinst script for Lyrion Music Server
+
+set -e
+. /usr/share/debconf/confmodule
+
+case "$1" in
+    configure)
+	# we're going to replace logitechmediaserver with lyrionmusicserver
+	if [ -f /etc/init.d/logitechmediaserver ]; then
+		/etc/init.d/logitechmediaserver stop
+	fi
+
+	# move some leftovers
+	if [ -f /etc/default/logitechmediaserver ]; then
+		if [ -f /etc/default/lyrionmusicserver ]; then
+			rm -f /etc/default/lyrionmusicserver
```

Comment:
Isn't this superfluous, as the `mv` below will overwrite it anyways when it exists?

Reply:
It would lead orphaned files behind, or files which aren't in the new folder, wouldn't it?

But TBH: I haven't really touched any of this beyond replacing the names. The file is flagged as new, but only has been renamed. Don't know why for two of them this is not recognized.

Reply:
Probably because too much of the content is changed?